core/src/main/java/com/uber/m3/tally/ScopeImpl.java

```diff
@@ -166,10 +166,16 @@ String fullyQualifiedName(String name) {
      * Returns a {@link Snapshot} of this {@link Scope}.
      * @return a {@link Snapshot} of this {@link Scope}
      */
```

Comment:
could you please update the java doc here mentioning that this method is meant for testing purposes only and should not be used in production?